compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/AMDAIEOps.cpp

```diff
@@ -93,6 +90,70 @@ TileOp CoreOp::getTileOp() {
 // AMDAIE_DmaCpyNdBaseOp
 //===----------------------------------------------------------------------===//
 
+namespace {
+// Simplified from upstream MLIR's foldDynamicIndexList:
```

Comment:
```suggestion
/// Simplified from upstream MLIR's foldDynamicIndexList:
```
Nit, but IREE uses `///` for function/class comments.